lib/pliny/error_reporter.rb

```diff
@@ -14,29 +13,35 @@ def self.notify(exception, context: {}, rack_env: {})
     end
   end
 
-  class RollbarReporter
-    include ::Rollbar::ExceptionReporter
-    include ::Rollbar::RequestDataExtractor
+  begin
```

Comment:
I'm personally not super crazy about rescuing `LoadError` on require attempts. I wonder if we can make this conditional based on the current error reporting configuration?